### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by maury18**
in this file, the most important change is that I added media queries for a 1024px screen (desktop screen) so it is responsive for the mobile screen (375px screen) and desktop screen(1024px screen) besides that I made some changes for the header background and cleared some old code.

# Changes:
- Responsive for desktop screen(1024px).
- changes in backgrounds (moved the icons with position: absolute).
- Cleared old code

# Solved the mobile and desktop responsive error.
- It was a margin problem that made it scroll to the right side but it's fixed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,208 +15,416 @@
...
+                    <img src="./img/desktop-version/Ellipse5.svg" alt="" class="image5">
+                    <img src="./img/desktop-version/Ellipse3.svg" alt="" class="image6">
+                    <img src="./img/desktop-version/Ellipse12.svg" alt="" class="image7">
+                    <img src="./img/desktop-version/Ellipse13.svg" alt="" class="image8">
+                    <img src="./img/desktop-version/Ellipse4.svg" alt="" class="image9">
+                    <img src="./img/desktop-version/Rectangle5.svg" alt="" class="image10">
+                    <img src="./img/desktop-version/Ellipse8.svg" alt="" class="image11">
+                    <img src="./img/desktop-version/Ellipse8.svg" alt="" class="image12">
+                    <img src="./img/desktop-version/Ellipse8.svg" alt="" class="image13">
+                </div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly remove the div of images and instead use background-image in CSS you can check this [link](https://www.w3schools.com/css/css3_backgrounds.asp) if you want to add multiple background images.
- [ ] As you can see that it has made the design in the mobile version doesn't look responsive.
Screenshot from 2022-04-28 23-12-22

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @maury18 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # My-Portfolio
 
-> This is an HTML project about my portfolio, here comes the first 2 sections, then the projects section where you can see a few projects,about me section and the form section.
+> This is an HTML project about my portfolio, here comes the first 2 sections, then the projects section where you can see a few projects,about me section and the form section, this porject is responsive for a 375px screen(mobile screen) and a 1021px screen(desktop screen).
```
Comment (rustam-tolipov) [inline README.md:3]: - [ ] [OPTIONAL] MIT.md license file is missing so please add it 📃
- [ ] [OPTIONAL] I also recommend adding a screenshot to the readme file 📷
- [ ] [OPTIONAL] Let's also remove the unused parts from the readme like _Prerequisites, Setup, Install, Usage, Run tests, Deployment_

**Review by rustam-tolipov — APPROVED** [review]
## Hello @maury18 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**